### ISSUE TITLE
Nav Menu Dropdown [aria-hidden="true"] elements contain focusable descendants

### ISSUE THREAD

**Issue by shimsa** [elementor/elementor#11537]
## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest stable version of Elementor.

## Description

When using Nav Menu Dropdown, web.dev/measure tool based on Google Lighthouse detects "[aria-hidden="true"] elements contain focusable descendants"

![nav-menu-aria-hidden-true](https://user-images.githubusercontent.com/7617350/83768036-9ab71000-a67e-11ea-88e9-dbd37acf7346.png)

## Steps to reproduce

- create Nav Menu with Dropdown
- go to https://web.dev/measure/
- type your domain and run audit
- in accessibility category is this problem "`[aria-hidden="true"]` elements contain focusable descendents"
- click on View report for more informations

## Isolating the problem

- [x] This bug happens with only Elementor plugin active (and Elementor Pro).
- [x] This bug happens with a default WordPress theme active.
- [x] I can reproduce this bug consistently using the steps above.

**Comment by tbusillo** [comment]
I don't think this is an actual accessibility issue and is a false positive in Lighthouse. The elements in the drop-down _are_ hidden, until the user activates the menu. Once the drop-down menu is triggered, the `aria-hidden` attribute is changed to "false", which is the correct behavior since they are now visible and focusable. If you try actually tabbing through while the drop-down is collapsed, the child elements do not receive focus.

**Comment by draconnia** [comment]
I second this, same problem with Google Lighthouse

**Comment by AngelDeLeon96** [comment]
I have the same problem

**Comment by mcrowell** [comment]
+1 I get the same results in Lighthouse.

**Comment by sybilrondeau** [comment]
+1

**Comment by joe-hastings** [comment]
@KingYes What was done to resolve this?

**Comment by Flexiscreens** [comment]
I have this issue in Lighthouse as well. The menu works correctly, but the issue relates to accessibility, not normal use.
Also interested to know why it was closed. Do new comments re-open a thread?
